src/open_inwoner/openzaak/tests/mocks.py

```diff
@@ -28,9 +28,30 @@ def __init__(self):
             ],
         }
 
+        self.task1 = {
```

Comment:
Would it be a good idea to split this mock data class? These tasks are not related to the open submissions. We could have a class ESuiteSubmissionData and a ESuiteTaskData: